.github/workflows/ci.yaml

```diff
@@ -40,25 +40,47 @@ jobs:
           key: docker-cache-${{ runner.os }}-${{ github.sha }}
           restore-keys: docker-cache-${{ runner.os }}-
 
-      - name: Install Aqua
+      # Install Aqua
+      - name: Install Aqua on Linux/macOS
         uses: aquaproj/aqua-installer@e2d0136abcf70b7a2f6f505720640750557c4b33 # v3.1.1
         with:
           aqua_version: v2.44.1
+          
+      - name: Set aqua path on Linux/macOS
+        run: |
+          echo "${AQUA_ROOT_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/aquaproj-aqua}/bin" >> $GITHUB_PATH
 
+      # Install tools
       - name: Install tools
-        run: aqua install
-      
-      - name: Install Windsor CLI from Main Branch
         run: |
-          git clone https://github.com/windsorcli/cli.git
-          cd cli/cmd/windsor
-          go build -o windsor
-          chmod +x windsor
-          mkdir -p $HOME/.local/bin
-          mv windsor $HOME/.local/bin
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-
-      - name: Print Windsor Version
+          aqua install     
+
+      # Install Windsor CLI
+      - name: Install Windsor CLI
+        uses: windsorcli/action/.github/actions/windsorcli@078842a08b468bd2278563bbc929e9d968095988 # v0.2.0
+        with:
+          version: ${{ env.windsorcli_version }}
+          install_folder: ${{ github.workspace }}/bin
+          context: local
+
+      # Windsor Up
+      - name: Windsor Up on Linux/macOS
         run: |
-          windsor version
-          
\ No newline at end of file
+          # Set WINDSOR_PROJECT_ROOT
+          export WINDSOR_PROJECT_ROOT="${{ github.workspace }}"
+
+          # Windsor Init
+          windsor init local
+
+          # Windsor Up
+          windsor context get
+          windsor up --install --verbose
+
+      # Windsor Down
+      - name: Windsor Down on Linux/macOS
+        run: |
+          # Set WINDSOR_PROJECT_ROOT
+          export WINDSOR_PROJECT_ROOT="${{ github.workspace }}"
```

Comment:
I think you can just set this once at the top of the action and it will set it everywhere, you don't have to set it in every step.